### PR TITLE
Add GDoc information to CII Metric

### PR DESCRIPTION
**Pull request by Nebrethar**
PR to add CII information from the google doc

Signed-off-by: Matt Snell <msnell@unomaha.edu>

### CODE REVIEW

**Comment by GeorgLink** [comment]
could you include a link to "the google doc"?

**Comment by Nebrethar** [comment]
> Could you please include a link to "the google doc"?

https://docs.google.com/document/d/1canGZZYi5iIYRU6qRMPDppfRIUlPSHlzjB5MCofbuf8/edit?usp=sharing

Thanks for reminding me!

**Comment by sgoggins** [comment]
@germonprez or @klumb : Could you look this over to see how close it is to the standard structure that we currently are working with. I know we changed it in the past few weeks, and I am not sure if this is "it". :)

File: metrics/CII_Best_Practices.md [hunk 1]
```diff
@@ -1,47 +1,97 @@
...
+The input is an API call; GET request to :
+```
+https://bestpractices.coreinfrastructure.org/projects.json?pq= + ciiURL
+```
+Where ciiURL is the URL that CII has on file for the project. This value is often a GitHub URL
 
-## 7. Test Cases (Examples)
-Sample inputs (including contexts) and expected outputs for this metric. Implementers can test their implementations against these test cases. For quantitative metrics, this could include a static repository with known metric results, or just inputs and output. For qualitative metrics, this may be more difficult.
+The output is a set of values pertaining to CII pest practices.
+Here is a snippet of values from an API call regarding box/bart:
```
Comment (sgoggins) [inline metrics/CII_Best_Practices.md]: Maybe we want to know what the "full load" looks like? Reference another document?

Comment (sgoggins) [reply metrics/CII_Best_Practices.md]: @Nebrethar : Maybe just a link out per our call discussion / Risk Working group.

**Comment by klumb** [comment]
I think this is where we are at

**Description (required)**
**Objectives (required)**
**Implementation (required - sub-headings optional)**
- Filters (optional)
- Visualizations (optional)
- Tools providing metric (was: known or sample implementations) (optional)
- Data Collection Strategies (optional)
- Success Metrics (optional) -- maybe reword this (To be discussed in the D&I WG)
**Resources (required)**

**Comment by Nebrethar** [comment]
The CII metric is now updated to the format @klumb provided. I also made some minor improvements to grammar, etc.
